Dynavity/Dynavity/view/CanvasSelectionView.swift

```diff
@@ -224,4 +228,12 @@ extension CanvasSelectionView {
         errorAlert.addAction(UIAlertAction(title: "Dismiss", style: .cancel, handler: nil))
         self.showAlert(alert: errorAlert)
     }
+
+    private func unableToLoadAnnotationsHandler() {
```

Comment:
This function isn't used anywhere right? Is this intentional?

Reply:
Nope not intentional, missed this one out, thanks!